### PR TITLE
Support Flink 1.11 and 1.12

### PR DESCRIPTION
**Pull request by humengyu2012**

### CODE REVIEW

File: flink/flink-1.11/src/main/java/org/tikv/bigdata/flink/tidb/TiDBCatalog.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ * limitations under the License.
+ */
+
+package org.tikv.bigdata.flink.tidb;
+
+import java.util.Map;
+import java.util.Optional;
+import org.apache.flink.table.factories.Factory;
+
+public class TiDBCatalog extends TiDBBaseCatalog {
```
Comment (marsishandsome) [inline flink/flink-1.11/src/main/java/org/tikv/bigdata/flink/tidb/TiDBCatalog.java:23]: `TiDBCatalog` is same as `TiDBBaseCatalog`, maybe we do not need `TiDBBaseCatalog`.

**Review by marsishandsome — APPROVED** [review]
LGTM

**Review by sunxiaoguang — APPROVED** [review]
LGTM

**Pull request merged**